prow/plugins/trigger/ic_test.go

```diff
@@ -174,14 +282,14 @@ func TestHandleIssueComment(t *testing.T) {
 					Name:      "job",
 					AlwaysRun: true,
 					Context:   "pull-job",
-					Trigger:   "@k8s-bot test this",
+					Trigger:   "(^@k8s-bot test this\r?$|^/test\r?$)",
```

Comment:
i will remove the unnecessary `|`, and add more test cases to handle corner cases. The suffix `\r?` does handle white spaces at end of line. i will double check the cases such as `/test\r\n flake ref:#999999`.